packages/terra-button/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Directly include packaged themes
```

Comment:
should we add consume terra-theme-context to these?

Reply:
Yeah I need to update those.